service/common/src/main/java/org/apache/polaris/service/exception/IcebergExceptionMapper.java

```diff
@@ -68,6 +68,8 @@ public class IcebergExceptionMapper implements ExceptionMapper<RuntimeException>
   /** Signifies that we could not extract an HTTP code from a given cloud exception */
   public static final int UNKNOWN_CLOUD_HTTP_CODE = -1;
 
+  @VisibleForTesting public static final String AZURE_STORAGE_URL_SUFFIX = ".blob.core.windows.net";
```

Comment:
nit: This is not the only possible Azure suffix, AFAIK

Reply:
I do not recall all :sweat_smile: IIRC, there was something like `.dfs.windows.net`

Reply:
Updated to just `windows.net` for now. We can also do this iteratively if we discover more. I found this [official list](https://learn.microsoft.com/en-us/azure/security/fundamentals/azure-domains) from Microsoft which for some reason doesn't include `.dfs.windows.net` even though Google suggests that one might actually exist.